Dockerfile

```diff
@@ -115,6 +115,8 @@ FROM resin/i386-node:6.5-slim as i386-node-base
 RUN echo '#!/bin/sh\nexit 0' > /usr/bin/cross-build-start && chmod +x /usr/bin/cross-build-start \
```

Comment:
@pcarranzav why are you doing those two things here and later you do `RUN [ "cross-build-start" ]`?

Reply:
@petrosagg since the Dockerfile is the same for all architectures, and the ARM builds need a cross-build-start, we add a cross-build-start to i386 and amd64 that is a noop. We can't conditionally run cross-build-start because checking for the condition would need to be done in qemu (discussed in https://github.com/resin-io/resin-supervisor/pull/470#discussion_r127535215 ) - but if you have any other ideas I'd love to hear them.